generators/entity-server/templates/src/test/java/package/web/rest/EntityResourceIT.java.ejs

```diff
@@ -824,166 +818,164 @@ class <%= entityClass %>ResourceIT <% if (databaseType === 'cassandra') { %>exte
-            <%_ if ((field.fieldType === 'byte[]' || field.fieldType === 'ByteBuffer') && field.fieldTypeBlobContent !== 'text') { _%>
+  <%_ if (field.fieldTypeBinary && !field.blobContentTypeText) { _%>
             <%= !reactive ? '.andExpect(' : '.' %>jsonPath("$.[*].<%= field.fieldName %>ContentType").value(hasItem(<%= 'DEFAULT_' + field.fieldNameUnderscored.toUpperCase() %>_CONTENT_TYPE))<%= !reactive ? ')' : '' %>
-            <%_ } _%>
+  <%_ } _%>
             <%= !reactive ? '.andExpect(' : '.' %>jsonPath("$.[*].<%= field.fieldName %>").value(hasItem(<%
-                    if ((field.fieldType === 'byte[]' || field.fieldType === 'ByteBuffer') && field.fieldTypeBlobContent !== 'text') { %>Base64Utils.encodeToString(<% } else
-                        if (field.fieldType === 'ZonedDateTime') { %>sameInstant(<% } else
-                            if (field.fieldType === 'BigDecimal') { %>sameNumber(<% } %><%= 'DEFAULT_' + field.fieldNameUnderscored.toUpperCase() %><%
-                        if ((field.fieldType === 'byte[]' || field.fieldType === 'ByteBuffer') && field.fieldTypeBlobContent !== 'text') { %><%
-                            if (databaseType === 'cassandra') { %>.array()<% } %>)<% } else
-                                if (field.fieldType === 'Integer') { %><% } else
-                                    if (field.fieldType === 'Long') { %>.intValue()<% } else
-                                        if (field.fieldType === 'Float' || field.fieldType === 'Double') { %>.doubleValue()<% } else
-                                            if (field.fieldType === 'BigDecimal') { %>)<% } else
-                                                if (field.fieldType === 'Boolean') { %>.booleanValue()<% } else
-                                                    if (field.fieldType === 'ZonedDateTime') { %>)<% } else
-                                                        if (field.fieldType !== 'String') { %>.toString()<% } %>))<%= !reactive ? ')' : '' %><%_ } _%>;
+  if (field.fieldTypeBinary && !field.blobContentTypeText) { %>Base64Utils.encodeToString(<% } else
+  if (field.fieldTypeZonedDateTime) { %>sameInstant(<% } else
+  if (field.fieldTypeBigDecimal) { %>sameNumber(<% } %><%= 'DEFAULT_' + field.fieldNameUnderscored.toUpperCase() %><%
+  if (field.fieldTypeBinary && !field.blobContentTypeText) { %><%
+    if (databaseTypeCassandra) { %>.array()<% } %>)<% } else
+  if (field.fieldTypeInteger) { %><% } else
+  if (field.fieldTypeLong) { %>.intValue()<% } else
+  if (field.fieldTypeFloat || field.fieldTypeDouble) { %>.doubleValue()<% } else
+  if (field.fieldTypeBigDecimal) { %>)<% } else
+  if (field.fieldTypeBoolean) { %>.booleanValue()<% } else
+  if (field.fieldTypeZonedDateTime) { %>)<% } else
+  if (!field.fieldTypeString) { %>.toString()<% } %>))<%= !reactive ? ')' : '' %><%_ } _%>;
```

Comment:
Maybe this is easier to understand:
```suggestion
  if (field.fieldTypeBinary && !field.blobContentTypeText) {
    %>Base64Utils.encodeToString(<%
  } else if (field.fieldTypeZonedDateTime) {
    %>sameInstant(<%
  } else if (field.fieldTypeBigDecimal) {
    %>sameNumber(<%
  }
  %><%= 'DEFAULT_' + field.fieldNameUnderscored.toUpperCase() %><%
  if (field.fieldTypeBinary && !field.blobContentTypeText) {
    if (databaseTypeCassandra) { %>.array()<% }
    %>)<%
  } else if (field.fieldTypeInteger) {
    %><%
  } else if (field.fieldTypeLong) {
    %>.intValue()<%
  } else if (field.fieldTypeFloat || field.fieldTypeDouble) {
    %>.doubleValue()<%
  } else if (field.fieldTypeBigDecimal) {
    %>)<%
  } else if (field.fieldTypeBoolean) {
    %>.booleanValue()<%
  } else if (field.fieldTypeZonedDateTime) {
    %>)<%
  } else if (!field.fieldTypeString) {
    %>.toString()<%
  }
  %>))<%= !reactive ? ')' : '' %><%_
} _%>;
```

Reply:
More understandable from a Jhipster developper yes, but you'll have a linebreak on the generated code

Reply:
Every line break added is inside a js tag, so the generated source would not change.

It’s just a suggestion anyway.